src/main/java/xyz/nkomarn/harbor/provider/DefaultAFKProvider.java

```diff
@@ -0,0 +1,85 @@
+package xyz.nkomarn.harbor.provider;
+
+import org.bukkit.Bukkit;
+import org.bukkit.configuration.ConfigurationSection;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerQuitEvent;
+import org.jetbrains.annotations.NotNull;
+import xyz.nkomarn.harbor.Harbor;
+import xyz.nkomarn.harbor.api.AFKProvider;
+import xyz.nkomarn.harbor.listener.AfkListeners;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class DefaultAFKProvider implements AFKProvider, Listener {
```

Comment:
this class doesnt contain any event listener, but is a listener.
It should be only an AFKProvider.

Reply:
I'm slightly confused as it has an event handler, is that not a listener?
![image](https://user-images.githubusercontent.com/4931930/112874637-65056c80-9080-11eb-8ace-df8651c7d56b.png)

Reply:
Well... Seems like I overlooked this xD Then ignore this I guess xD